src/util/parseLink.js

```diff
@@ -0,0 +1,12 @@
+const linkRegex = /^<?(https?:\/\/.+?)>?$/;
+
+function parseLink(url)
+{
+	if(!url) return url;
+	const matches = linkRegex.exec(url);
+	if(!matches) return undefined;
+	const link = matches[1];
+	return link;
+}
+
+module.exports = parseLink;
```

Comment:
```suggestion
module.exports = parseLink;

```
End files with newline